docs/gelly_guide.md

```diff
@@ -430,6 +440,46 @@ public static final class Messenger extends MessagingFunction {...}
 
 {% endhighlight %}
 
+The following example illustrates the usage of the degree, the number of vertices as well as the messaging direction options.
+
+{% highlight java %}
+
+Graph<Long, Double, Double> graph = ...
+
+// configure the iteration
+IterationConfiguration parameters = new IterationConfiguration();
+
+// set the messaging direction
+parameters.setDirection(EdgeDirection.IN);
```

Comment:
This is not used anywhere in the example. I would remove it from here and add another example only for the direction, together with the figures I mention above.